rebar.config

```diff
@@ -1,6 +1,6 @@
 {erl_opts, [debug_info]}.
 
-{deps, [{chatterbox, {pkg, ts_chatterbox}},
+{deps, [{chatterbox, ".*", {git, "https://github.com/novalabsxyz/chatterbox", {branch, "adt/perf-nodebug"}}},
```

Comment:
Switch this to `tsloughter/chatterbox` and I'll merge after the rebase. Not ready to make a chatterbox release with such big changes yet but want it easy to test it for others.